### DISCUSSION TITLE
Sync the instance

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1744] [Help] [answered]
Hey, when Im changing instance, dying, and changing gamemode/team, the player moves a few blocks.![image-4.png](https://user-images.githubusercontent.com/71212639/218027294-5b00a45c-1ec1-4f77-9c43-ff18b5792b36.png)

After a certain time (30 seconds or something), the player instance got synced, and the player is on the right position.

Is there something to make the instance sync like every 3 seconds?

**Comment by FluxCapacitor2** [comment, marked as answer]
You could try periodically calling [`askSynchronization`](https://javadoc.minestom.net/net/minestom/server/entity/Entity.html#askSynchronization()) on the player; this will call the [`synchronizePosition`](https://github.com/Minestom/Minestom/blob/84f4ae040e6a88e204f2c5a2dffb03d0c3a5c757/src/main/java/net/minestom/server/entity/Entity.java#L1556) method on the next tick, sending a teleport packet to viewers.

(The default synchronization period is once per minute, and it currently can't be changed with a system property)

**Reply by DevBlokJochem** [reply]
public void setCustomSynchronizationCooldown(@Nullable Duration cooldown) {
        this.customSynchronizationCooldown = cooldown;
    }
this will change the "once per minute"
